starport/services/scaffolder/scaffolder.go

```diff
@@ -62,13 +62,17 @@ func AddressPrefix(prefix string) Option {
 
 // New initializes a new Scaffolder for app at path.
 func New(path string, options ...Option) (*Scaffolder, error) {
+	path, err := filepath.Abs(path)
+	if err != nil {
+		return nil, err
+	}
+
 	s := &Scaffolder{
 		path:    path,
```

Comment:
do we need to keep this field anymore?

Reply:
we still use it in [some methods](https://github.com/tendermint/starport/pull/1518/files#diff-9523918b7613cca4a039caac14dfd792f62ce05849b77f862cc06b7e62d3025dR56
)

But we can base it as a parameter like a [module name and the message name](https://github.com/tendermint/starport/pull/1518/files#diff-9523918b7613cca4a039caac14dfd792f62ce05849b77f862cc06b7e62d3025dR56)